### PR TITLE
Add optional params to ModeratingSubreddit fetch

### PR DESCRIPTION
**Pull request by birakattack**
Add optional params for ModeratingSubreddit in order to retrieve the subreddit settings.

Add wikiMode, wikiEditAge, and wikiEditKarma properties to the api response mapping.

:eyeglasses: @schwers

### CODE REVIEW

File: collections/SubredditLists.js [hunk 1]
```diff
@@ -44,6 +44,19 @@ export class SubscribedSubreddits extends SubredditList {
...
 export class ModeratingSubreddits extends SubredditList {
   static sort = 'mine/moderator';
+
+  static async fetch(apiOptions, options={}, all=true) {
+    const { get } = SubredditEndpoint;
+    console.log({...this.baseOptions(apiOptions), ...options });
+    const allMergedSubreddits = await fetchAll(get, apiOptions, {
+      ...this.baseOptions({
+        ...apiOptions,
+        'view': 'mod'
```
Comment (schwers) [inline collections/SubredditLists.js]: `baseOptions` won't use `view` anymore, should this be `sort` or be one-level out in nesting (this way you could merge it onto the result of `baseOptions`)?. If it's `sort` you could change the static property

File: collections/SubredditLists.js [hunk 2]
```diff
@@ -44,6 +44,19 @@ export class SubscribedSubreddits extends SubredditList {
 
 export class ModeratingSubreddits extends SubredditList {
   static sort = 'mine/moderator';
+
+  static async fetch(apiOptions, options={}, all=true) {
+    const { get } = SubredditEndpoint;
+    console.log({...this.baseOptions(apiOptions), ...options });
```
Comment (schwers) [inline collections/SubredditLists.js]: Please remove

**Comment by schwers** [comment]
💅

**Comment by birakattack** [comment]
This is now an erroneous PR.

**Pull request closed without merging**